app/assets/javascripts/Components/Modals/submission_url_submit_modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Modal from "react-modal";
+
+class SubmitUrlUploadModal extends React.Component {
+  state = {
```

Comment:
State should be set in the `constructor` not as a static attribute on the class.